### ISSUE TITLE
Documentation-functionality conflict

### ISSUE THREAD

**Issue by sir-dunxalot** [knownasilya/ember-cli-toggle#17]
The documentation says that adding `off` and `on` to a toggle should not change the value of a bound property unless you 'overload' the label/value:

> Finally, it is sometimes the case that rather than a true or false value the toggle is meant to move between two discrete, but non-boolean states. In order to support this functionality there is an overloaded form of setting the on and off properties which not only sets a "label" for the state but also a "value". In our "hey" versus "ho" example you might do the following:

Thus, it is expected that `controller.toggled` is a boolean value equal to true or false in the following situation:

```
{{x-toggle value=controller.toggled showLabel='true' off='Hey' on='Ho'}}
```

However, in v0.1.15 of ember-cli-toggle, `controller.toggled` is always equal to `Ho` or `Hey` when it *should* be equal to `true` or `false`, respectively.

**Comment by knownasilya** [comment]
This is fixed in v0.5.5

**Comment by sir-dunxalot** [comment]
Great, thank you.
